l10n_ar_ux/views/account_move_view.xml

```diff
@@ -8,21 +8,13 @@
                 <field name="computed_currency_rate" string="Currency Rate (preview)" attrs="{'invisible': ['|', ('l10n_ar_currency_rate', '!=', 0.0), ('other_currency', '=', False)]}"/>
-                <button name="%(action_account_move_change_rate)d" type="action" icon="fa-pencil" style="color:red" class="btn-link" attrs="{'invisible':['|', '|', '|' , '|', ('state','!=','draft'), ('other_currency', '=', False), ('country_code', '!=', 'AR'), ('l10n_ar_currency_rate', '=', 0.0), ('l10n_ar_currency_rate', '=', False)]}" help="Currency rate forced"/>
-                <button name="%(action_account_move_change_rate)d" type="action" icon="fa-pencil" class="btn-link" attrs="{'invisible':['|', '|', '|' , ('state','!=','draft'), ('other_currency', '=', False), ('country_code', '!=', 'AR'), ('l10n_ar_currency_rate', '!=', 0.0)]}"/>
-            </button>
-            <field name="currency_id" position="after">
-                <field name="l10n_ar_currency_rate" attrs="{'invisible': ['|', ('country_code', '!=', 'AR'), '|', '|', ('l10n_ar_currency_rate', '=', False), ('l10n_ar_currency_rate', '=', 0.0), ('l10n_ar_currency_rate', '=', 1.0)]}"/>
-                <field name="computed_currency_rate" string="Currency Rate (preview)" attrs="{'invisible': ['|', ('country_code', '!=', 'AR'), '|', ('l10n_ar_currency_rate', '!=', 0.0), ('computed_currency_rate', '=', 1.0)]}"/>
-                <button name="%(action_account_move_change_rate)d" type="action" string="(change rate)" attrs="{'invisible':['|', ('country_code', '!=', 'AR'), '|', ('state','!=','draft'), ('computed_currency_rate', '=', 1.0)]}" class="oe_link"/>
-            </field>
-            <button name="%(account_ux.action_account_change_currency)d" position="attributes">
-                <attribute name="string">(change currency)</attribute>
+                <span groups="base.group_multi_currency" attrs="{'invisible': [('other_currency', '=', False)]}">)</span>
+                <!-- TODO arreglar que no se ve el cambio de color. Estos dos botones duplicados son solamente para mostrar cuando la tasa de cambio está forzada. Otra alternativa seria mandarlo en un warning arriba -->
+                <!-- <button name="%(action_account_move_change_rate)d" type="action" icon="fa-pencil" style="color:red" class="btn-link" attrs="{'invisible':['|', '|', '|' , '|', ('state','!=','draft'), ('other_currency', '=', False), ('country_code', '!=', 'AR'), ('l10n_ar_currency_rate', '=', 0.0), ('l10n_ar_currency_rate', '=', False)]}" title="Edit Rate"/>
+                <button name="%(action_account_move_change_rate)d" type="action" icon="fa-pencil" class="btn-link" attrs="{'invisible':['|', '|', '|' , ('state','!=','draft'), ('other_currency', '=', False), ('country_code', '!=', 'AR'), ('l10n_ar_currency_rate', '!=', 0.0)]}" title="Edit Rate"/> -->
```

Comment:
we are not showing any button to be able to change the rate. we can remove the color part if needed but need to show the button once

Reply:
not yet implemeneted